addons/pos_self_order/tests/test_frontend.py

```diff
@@ -39,3 +39,34 @@ def test_self_order_menu_only_accessing_without_token_tour(self):
             "self_order_menu_only_tour",
             login=None,
         )
+
+    def test_kiosk_order_tour(self):
+        self.pos_config_kiosk.with_user(self.pos_user).action_open_wizard()
+        self.start_tour(
+            self.pos_config_kiosk.self_order_kiosk_url,
+            "kiosk_order_tour",
+            login=None,
+        )
+        order_count = self.env['pos.order'].search_count([('tracking_number', '=', 'A1')])
+        self.assertEqual(order_count, 1)
+
+    def test_kiosk_order_table_service_tour(self):
+        self.pos_config_kiosk.self_order_kiosk_mode = "table"
+        self.pos_config_kiosk.with_user(self.pos_user).action_open_wizard()
+        self.start_tour(
+            self.pos_config_kiosk.self_order_kiosk_url,
+            "kiosk_order_tour_table_service",
+            login=None,
+        )
+        order_count = self.env['pos.order'].search_count([('tracking_number', '=', 'A1')])
+        self.assertEqual(order_count, 1)
+
+    def test_kiosk_order_takeaway_tour(self):
+        self.pos_config_kiosk.self_order_kiosk_mode = "table"
+        self.pos_config_kiosk.self_order_kiosk_takeaway = True
+        self.pos_config_kiosk.with_user(self.pos_user).action_open_wizard()
+        self.start_tour(
+            self.pos_config_kiosk.self_order_kiosk_url,
+            "kiosk_order_tour_takeaway",
+            login=None,
+        )
```

Comment:
the kiosk should also have a test for the attributes

Reply:
Make another task for that it came afterwards and we need to merge this asap